src/CONST.js

```diff
@@ -342,6 +347,9 @@ const CONST = {
             modifiers: [],
         },
     },
+    KEYBOARD_SHORTCUTS_TYPES: {
+        NAVIGATION_SHORTCUT: KEYBOARD_SHORTCUT_NAVIGATION_TYPE,
```

Comment:
```suggestion
        NAVIGATION_SHORTCUT: 'NAVIGATION_SHORTCUT',
```

I don't think we should define this first as a const at the top like `KEYBOARD_SHORTCUT_NAVIGATION_TYPE`. It should be easy to just do `KEYBOARD_SHORTCUTS_TYPES.NAVIGATION_SHORTCUT` where needed